templates/config.hocon.tmpl

```diff
@@ -1,36 +1,25 @@
-source = "kinesis"
-sink = "kinesis"
-aws {
-  accessKey = "iam"
-  secretKey = "iam"
-}
-kinesis {
-  initialPosition = "${initial_position}"
-  initialTimestamp = ""
-  maxRecords = 10000
-  region = "${region}"
-  appName = "${app_name}"
-  disableCloudWatch = true
-}
-streams {
-  inStreamName = "${in_stream_name}"
-  outStreamName = "${out_stream_name}"
-  buffer {
-    byteLimit = ${byte_limit}
-    recordLimit = ${record_limit}
-    timeLimit = ${time_limit_ms}
+{
+  "region": "${region}",
+  "purpose": "RAW",
```

Comment:
Should the purpose always be "RAW" ?  This should be configurable I would think depending on the source data type?

Reply:
I pondered this and decided to hard code it to RAW for simplicity after finding this comment in the example HOCON:

>   # RAW simply sinks data 1:1
>  # ENRICHED_EVENTS work with monitoring.statsd to report metrics (identical to RAW otherwise)
>  # SELF_DESCRIBING partitions self-describing data (such as JSON) by its schema

https://github.com/snowplow/snowplow-s3-loader/blob/master/config/config.hocon.sample#L5-L9

Since we're not enabling monitoring.statsd here, and I don't believe any SELF_DESCRIBING data (?) then I didn't see any time we'd set it to anything but RAW. If you can think of a legit use case for the other values though, or if it'll be valuable in the future, then happy to add a var for this.